javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+const $countDown = document.querySelector('.countdown-display');
+let count = $countDown.textContent;
+const interval = setInterval(countDown, 1000);
+
+function countDown() {
+  $countDown.textContent = count;
+
+  if (count === 0) {
+    $countDown.textContent = '~Earth Beeeeloooww Us~';
```

Comment:
If the `count` is 0, then you should only be setting the text content to the string on line 9. You shouldn't iniitally be setting it's `textContent` to the number, and then to the new string. It's one or the other